test/e2e/validation_test.go

```diff
@@ -36,6 +34,10 @@ import (
 	"github.com/openshift/windows-machine-config-operator/pkg/windows"
 )
 
+const (
```

Comment:
Will add a comment here in the next push

Reply:
Also consider naming it `wmcoContainerName` or similar as it is not clear which deployment is being referred to